email_tracking/sample.env

```diff
@@ -0,0 +1,2 @@
+EASYPOST_API_KEY=(get from https://www.easypost.com/account#/api-keys)
+SENDGRID_API_KEY=(get from https://app.sendgrid.com/settings/api_keys - if it doesnt already exist create one by clicking on create api key button on top right cornor)
```

Comment:
well I think all the sample env for all the other examples are in this format.. so just trying to keep it consistent.